ssl/ssl_sess.c

```diff
@@ -118,8 +113,9 @@ SSL_SESSION *SSL_SESSION_new(void)
 
     ss->verify_result = 1;      /* avoid 0 (= X509_V_OK) just in case */
     ss->references = 1;
-    ss->timeout = 60 * 5 + 4;   /* 5 minute timeout by default */
-    ss->time = time(NULL);
+   /* 5 minute timeout by default */
+    ss->timeout = ossl_seconds2time(60 * 5 + 4);
```

Comment:
Five minute 4 second timeout? :)

Reply:
That's what was there.  No idea why :confused: